src/test/rustdoc-json/impls/blanket_with_local.rs

```diff
@@ -4,10 +4,12 @@
 // @has blanket_with_local.json "$.index[*][?(@.name=='Load')]"
 pub trait Load {
     fn load() {}
+    fn write(self) {}
```

Comment:
Since you added this method, shouldn't there be a matching check?

Reply:
Should probably add a check for both the methods. The checks are more to ensure the methods *exist* in the output, so we know we're testing the ICE correctly.